mmeds/database/database.py

```diff
@@ -897,6 +897,16 @@ def clear_mongo_data(self, username):
         for doc in data:
             doc.delete()
 
+    @classmethod
+    def get_study_from_access_code(cls, code):
+        """ For server use """
+        return MMEDSDoc.objects(access_code=code)[0]
```

Comment:
```suggestion
        return MMEDSDoc.objects(access_code=code).first()
```
MMEDSDoc.objects() Returns an iterator not a fully instantiated list so `first()` is more correct